core/src/test/scala/integration/kafka/admin/RemoteTopicCrudTest.scala

```diff
@@ -409,10 +438,15 @@ class RemoteTopicCrudTest extends IntegrationTestHarness {
             topicConfig.getProperty(TopicConfig.RETENTION_BYTES_CONFIG).toLong ==
               logBuffer.head.config.retentionSize
         }
-        if (topicConfig.contains(TopicConfig.REMOTE_LOG_DISABLE_POLICY_CONFIG)) {
+        if (topicConfig.contains(TopicConfig.REMOTE_COPY_DISABLED_CONFIG)) {
+          result = result &&
+            topicConfig.getProperty(TopicConfig.REMOTE_COPY_DISABLED_CONFIG).equals(
```

Comment:
we are comparing the string vs boolean, please change it to:
```
topicConfig.getProperty(TopicConfig.REMOTE_COPY_DISABLED_CONFIG).toBoolean ==
              logBuffer.head.config.remoteCopyDisabled()
```

similarly for the  below `remoteLogDeleteOnDisable()`